atm.js

```diff
@@ -1,21 +1,101 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+function Account(value){
+  this.entries = [];
+  this.els = {
+    input: $("input[type=text]").eq(value),
+    deposit: $('input[type=button][value="Deposit"]').eq(value),
+    withdraw: $('input[type=button][value="Withdraw"]').eq(value),
+    total: $(".balance").eq(value)
+  };
 }
 
-function checkingWithdrawal(amount){
-
-}
+Account.prototype.calculateBalance = function(){
+  var total = 0;
+  for( var i = 0; i < this.entries.length; i++ ){
+    total += this.entries[i];
+  }
+  return total;
+};
 
-function savingsDeposit(amount){
+var checking = new Account(0);
+var savings = new Account(1);
```

Comment:
clever! you could also pass in a selector to specify which account you want, e.g:

```js
var checking = $(".account").eq(0);
new Account( checking )
```